models/quoteCollection.js

```diff
@@ -75,7 +74,7 @@ Object.assign(quoteCollection.methods, {
   getCollaborators() {
     const privateData = ['password'];
     const User = mongoose.model('User');
-    return Promise.all(this.collaborators.map(id => User.findById(id).omit(privateData)));
+    return User.find({ _id: { $in: this.collaborators } }).omit(privateData);
```

Comment:
It would be nice to create some sort of helper:
```js
Model.findManyById(arrayOfIds)
```
so you can write stuff like:
```js
Quote.findManyById(this.quotes);
```
and:
```js
User.findManyById(this.collaborators)...
```